README.md

```diff
@@ -53,26 +71,37 @@ INFO[0000] Profiling ...
 $ go get github.com/uber/go-torch
 ```
 
-Install the dependencies:
+### Install the Go dependencies:
 
 ```
 $ go get github.com/tools/godep
 $ godep restore
 ```
 
+### Get the flame graph script:
+
+```
+$ git clone https://github.com/brendangregg/FlameGraph.git
+```
+
 ## Integrating With Your Application
 
-Expose a pprof endpoint. Official Go docs are [here](https://golang.org/pkg/net/http/pprof/). If your application is already running a server on the DefaultServeMux, just add this import to your application.
+Expose a pprof endpoint. Official Go docs are
+[here](https://golang.org/pkg/net/http/pprof/). If your application is already
+running a server on the DefaultServeMux, just add this import to your
+application.
 
 ```go
-import _ "net/http/pprof"
+import _ "net/http/pprof" 
```

Comment:
it looks like you're adding a bunch of trailing spaces in this diff. is github lying to me?